python/enthic/download_from_INPI.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+========================================
+Download daily zip files from INPI's FTP
+========================================
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+
+from os.path import dirname, join, basename
+from os import remove
+import subprocess
+from json import load
+from ftplib import FTP_TLS
+from logging import info
+
+from enthic.extract_bundle import process_daily_zip_file
+
+
+################################################################################
+# READ CONFIGURATION
+with open(join(dirname(__file__), "configuration.json")) as json_configuration_file:
+    CONFIG = load(json_configuration_file)
+
+def add_data_to_database(file_path):
+    """
+        Add data contained in the given file into MySQL database
+        then delete temporary files if it's a success
+
+        :param file_path: daily INPI file to add to database
+    """
+    # extract data from xml files to csv files
+    process_daily_zip_file(file_path)
+
+    # Load new data into database
+    result = subprocess.run(["../sh/database-filling.sh", CONFIG["mySQL"]["enthic"]["password"]], stdout=subprocess.PIPE, check=True)
```

Comment:
Could make it a bit more safer using join(dirname(__file__) like above?

Reply:
I'm not sure that `join("../sh","database-filling.sh")` is more secure than hardcoded path `"../sh/database-filling.sh"`
Do you had something else in mind? Am I wrong?